src/Executables/PrintMinimumGridSpacing/PrintMinimumGridSpacing.cpp

```diff
@@ -0,0 +1,154 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/program_options.hpp>
+#include <iomanip>
+
+#include "Domain/InitialElementIds.hpp"
+#include "Domain/MinimumGridSpacing.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Parallel/Printf.hpp"
+#include "Options/ParseOptions.hpp"
+
+using frame = Frame::Inertial;
+
+class DimensionOption;
+struct Dimension {
+  using type = DimensionOption;
+  static constexpr OptionString help = {"help"};
+};
+
+class DimensionOption {
```

Comment:
This class isn't used

Reply:
This class is used to mark the `type` of the `Dimension` struct used to create an option group - in short, to make the dimension an option to be parsed from the yaml file. I followed the option parsing guide here: https://spectre-code.org/dev_guide_option_parsing.html

D you mean that making the class isn't necessary to create an option to represent the dimension?

Reply:
The option handling infrastructure has recently changed, anyway... @macedo22 you can see the newest way to do it at  https://spectre-code.org/dev_guide_parallelization_foundations.html